pkg/chains/formats/intotoite6/intotoite6_test.go

```diff
@@ -67,18 +68,18 @@ func TestCreatePayload1(t *testing.T) {
 			},
 			Invocation: slsa.ProvenanceInvocation{
 				Parameters: map[string]string{
-					"IMAGE":             "{string test.io/test/image []}",
-					"CHAINS-GIT_COMMIT": "{string abcd []}",
-					"CHAINS-GIT_URL":    "{string https://git.test.com []}",
-					"filename":          "{string /bin/ls []}",
+					"IMAGE":             "test.io/test/image",
+					"CHAINS-GIT_COMMIT": "abcd",
+					"CHAINS-GIT_URL":    "https://git.test.com",
+					"filename":          "/bin/ls",
```

Comment:
If we use MarshallJSON as below this needs to be changed:

```suggestion
					"IMAGE":             `"test.io/test/image"`,
					"CHAINS-GIT_COMMIT": `"abcd"`,
					"CHAINS-GIT_URL":    `"https://git.test.com"`,
					"filename":          `"/bin/ls"`,
```